app/web/src/store/asset.store.ts

```diff
@@ -155,24 +135,26 @@ export const useAssetStore = () => {
           this.openAssetFuncIds[assetId] = funcs;
         },
 
-        closeFunc(assetId: AssetId, funcId: FuncId) {
+        CLOSE_FUNC(assetId: AssetId, funcId: FuncId) {
```

Comment:
the convention here (which has already been broken in a few places) is that the ALL_UPPER functions are api action (ie they make an api request, and so will have a request status available).

Reply:
yes plz. lets try to keep the UPPER_CASE actions only things that call the api